domains/sirilalithaadapa.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "sirilalithaadapa personal developer website",
+    "repo": "https://github.com/sirilalithaadapa",
+    "owner": {
+        "username": "sirilalithaadapa",
+        "email": "sirilalitha9088@gmail.com"
+    },
+    "record": {
+        "CNAME": "pafg.com.bd",
+        "URL": "https://sirilalithaadapa.is-a.dev"
```

Comment:
This doesn't seem like a legitimate use for this subdomain. I'm closing this pr. If you meant this only as a placeholder, you can create a new pr with a github pages cname record or a domain you own.